### PR TITLE
Feature/dp 6516 bouwdossiers

### PR DESCRIPTION
**Pull request by chrisvanmook**

### CODE REVIEW

File: modules/detail/components/partial-select/partials/bag/verblijfsobject.html [hunk 1]
```diff
@@ -117,4 +117,14 @@
             user="user"
         />
     </div>
+
+    <div class="grex-stadsdelen c-detail__block">
```
Comment (aditudorache) [inline modules/detail/components/partial-select/partials/bag/verblijfsobject.html]: please review the class name, this is nog grex-stadsdelen

File: modules/detail/components/partial-select/partials/bag/verblijfsobject.html [hunk 2]
```diff
@@ -117,4 +117,14 @@
...
+        <dp-glossary-header definition="BOUWDOSSIERS" use-plural="true"></dp-glossary-header>
+        <div ng-if="!user.scopes.includes('BD/R')"
+             ng-include="'modules/detail/components/detail/templates/grondexploitatie/unauthorized.html'">
+        </div>
+        <div ng-if="user.scopes.includes('BD/R')">
+            <dp-api-call no-result-message="Geen of nog geen bouwdossiers beschikbaar" endpoint="https://acc.api.data.amsterdam.nl/stadsarchief/bouwdossier/?{{subType}}={{id}}" partial="bouwdossiers" user="vm.user"></dp-api-call>
```
Comment (rmaijers) [inline modules/detail/components/partial-select/partials/bag/verblijfsobject.html]: a hard reference to acceptance api?

File: src/app/PiwikContext.jsx [hunk 3]
```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+const PiwikContext = React.createContext();
```
Comment (royderks) [inline src/app/PiwikContext.jsx]: MatomoContext

Comment (royderks) [reply src/app/PiwikContext.jsx]: File can be deleted?

**Pull request merged**